example_configs/ignite_2.6.0.yml

```diff
@@ -0,0 +1,95 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+#
+#
+lowercaseOutputLabelNames: true
+lowercaseOutputName: true
+rules:
+# Apache Ignite provide runtime information on a cluster.  Keep in mind that there will be a certain
+# network delay (usually equal to metrics update delay).
+#
+# see https://ignite.apache.org/releases/latest/javadoc/org/apache/ignite/cluster/ClusterMetrics.html
+- pattern: "^org.apache<clsLdr=(.+), group=Kernal, name=ClusterMetrics(.+)><>(HeapMemory.*):"
+  name: ignite_cluster_heap_$1
+  help: Ignite cluster amount of heap memory in bytes
+  labels:
+    attr: $3
+  type: GAUGE
+- pattern: "^org.apache<clsLdr=(.+), group=Kernal, name=ClusterMetrics(.+)><>(NonHeapMemory.*):"
+  name: ignite_cluster_nonheap_$1
+  help: Ignite cluster amount of non-heap memory in bytes
+  labels:
+    attr: $3
+  type: GAUGE
+- pattern: "^org.apache<clsLdr=(.+), group=Kernal, name=ClusterMetrics(.+)><>(.*Jobs):"
+  name: ignite_cluster_job_$1
+  help: Ignite cluster job detalization
+  labels:
+    attr: $3
+  type: COUNTER
+# Apache Ignite durable memory can be monitored via data region metrics which will help track overall
+# memory utilization and measure its cluster performance.
+#
+# see https://ignite.apache.org/releases/latest/javadoc/org/apache/ignite/mxbean/DataRegionMetricsMXBean.html
+- pattern: "^org.apache<clsLdr=(.+), group=DataRegionMetrics, name=(.+)><>(.*Pages|AllocationRate):"
+  name: ignite_dataregion_pages_$2_$1
+  help: Ignite durable memory pages currently loaded in RAM
+  labels:
+    attr: $3
+- pattern: "^org.apache<clsLdr=(.+), group=DataRegionMetrics, name=(.+)><>(Off[Hh]eap.*|MaxSize|PhysicalMemorySize|TotalAllocatedSize):"
```

Comment:
Are these the usual JVM memory metrics?

Reply:
Brian, no these metrics are collected through the whole cluster on specific `memory region`. They are not bounded to JVMs.